testdata/projects/gradle/.gradle/buildOutputCleanup/cache.properties

```diff
@@ -0,0 +1,2 @@
+#Wed May 08 15:41:54 IDT 2024
+gradle.version=7.6
```

Comment:
what are those files that were added in testdata/projectsgradle/.gradle dir?
I didn't see where you use a gradle project in this PR. Please just make sure those files were not added by mistake